index.html

```diff
@@ -35,8 +35,95 @@ <h2>I’m a software developer</h2>
                         <li><a href="#" class="fa fa-github-square"></a></li>
                         <li><a href="#" class="fa fa-instagram"></a></li>
                     </ul>
+            <h1>My Recent work</h1>
+    <div class="allw">
+        <article class="work1">
+            <div class="snapshot">
+                <img src="./Assets/Snapshoot.png" alt="">
+            </div>
+            <div>
+                <h3 class="multi">  Multi-Post Stories Gain+Glory</h3>
+            </div>
+            <ul class="lang">
+                <li class="l1"><a >Ruby on rails</a></li>
+                <li class="l2"><a >CSS</a></li>
+                <li class="l3"><a >JavaScript</a></li>
+                <li class="l4"><a >html</a></li>
+            </ul>
+            
+            
+        </article>
+        <button type="button" class="see">See projects</button>
+
+    </div>
+    <div class="allw">
+        <article class="work1">
+            <div class="snapshot">
+                <img src="./Assets/Snapshoot.png" alt="">
+            </div>
+            <div>
+                <h3 class="multi">  Multi-Post Stories Gain+Glory</h3>
+            </div>
+            <ul class="lang">
+                <li class="l1"><a >Ruby on rails</a></li>
+                <li class="l2"><a >CSS</a></li>
+                <li class="l3"><a >JavaScript</a></li>
+                <li class="l4"><a >html</a></li>
+            </ul>
+            
+            
+        </article>
+        <button type="button" class="see">See projects</button>
```

Comment:
Please make sure this button ("See project")  follow the button styling: Active, Hover and Pressed (see design)

![state](https://user-images.githubusercontent.com/26736582/205158449-84f90ca8-6ef6-4ec2-883c-9c5c8977c710.png)